### PR TITLE
Fix local variable shadowing warnings

### PR DESCRIPTION
**Pull request by slipher**
Later I will turn on the warning via the Daemon repo.

The bot vote kick one sadly conflicts with #2235.

### CODE REVIEW

**Comment by DolceTriade** [comment]
LGTM

**Review by necessarily-equal — APPROVED** [review]
LGTM

File: src/cgame/Filter.h [hunk 1]
```diff
@@ -52,9 +52,9 @@ class Filter
 	void Accumulate( int time, const T& sample )
 	{
 		samples.remove_if(
-			[&]( const std::pair<int,T>& sample )
+			[&]( const std::pair<int,T>& oldSample )
```
[the remark this answers is not in the document]
Comment (slipher) [reply src/cgame/Filter.h:55]: It's the lambda callback which receives an element of `samples`.

[the remark this answers is not in the document]
Comment (necessarily-equal) [reply src/cgame/Filter.h:55]: ok, fair enough

**Pull request merged**